src/pages/accomodation/components/ImagesList.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, ImageList, ImageListItem, useMediaQuery, useTheme } from '@mui/material';
+import ShowPhotos from '@src/components/full-view-accommodation/full-view-accommodation';
+import Slider from '@src/components/shared/Slider';
+import { AccommodationSingle } from '@src/types/accommodation.types';
+import { useState } from 'react';
+
+export default function ImagesList({ data }: { data: AccommodationSingle }) {
```

Comment:
Why do you pass accommodation here instead of list of media? Do you have any reason for that>

Reply:
because i need accommodationId inside showPhotos component which is inside imageList